scripts/generate_client_publish_job.py

```diff
@@ -42,7 +42,23 @@ def generate(integration_repo, args):
     stage_name = "trigger"
     document = {
         "stages": [stage_name],
-        "trigger:mender-qa": {
+    }
+
+    for integ_version in integration_versions_list:
+
+        subprocess.run(
+            ["git", "checkout", integ_version],
+            capture_output=True,
+            check=True,
+            cwd=integration_repo,
+        )
+
+        all_repos = subprocess.run(
+            [release_tool, "--list", "git"], capture_output=True, check=True
+        )
```

Comment:
I would have expected the `release_tool` to handle this, but OTOH it would maybe just do the same internally 🤷‍♂️